app/views/devise/shared/_error_messages.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if resource.errors.any? %>
-  <div id="error_explanation">
+  <div id="error_explanation" data-turbo-cache="false">
```

Comment:
Sorry, I'm not sure why this change is required. I couldn't find any strange behavior even if I remove this option. Could you explain a bad scenario when we remove it?

Reply:
It is not strictly required for the functionality. I was noticing an issue when going to edit the user, submitting a couple of times, then using the `Back` button (which is a javascript history.back() call), it was flashing the errors from the cached page before getting the response back from the server and displaying it:

https://user-images.githubusercontent.com/26328/217253701-6fa82752-f0b7-4503-b940-e061e087d526.mp4

So skipping the cache of this errors section seems to have "fixed" that for me. Makes sense?

Reply:
> it was flashing the errors

Thank you for the explanation. It's hard to notice but I understand🙂